untitled/test/com/aca/exam/exam2/FileSourceUserRepositoryTest.java

```diff
@@ -0,0 +1,63 @@
+package com.aca.exam.exam2;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+
+class FileSourceUserRepositoryTest {
+    private FileSourceUserRepository testSubject;
+
+    @BeforeEach
+    public void setUp() {
+        testSubject = FileSourceUserRepository.getInstance();
+    }
+
+    @Test
+    public void testGetInstance() {
+        Assertions.assertNotNull(testSubject);
+        FileSourceUserRepository fileSourceUserRepository = FileSourceUserRepository.getInstance();
+        Assertions.assertEquals(fileSourceUserRepository, testSubject);
+    }
+
+    @Test
+    public void testFindByIdWhenIdIsNotFound() {
```

Comment:
the test should assert the return result when the id is not null. The result should be null